doc/docs/1.11.x/deploy-manage/deploy/on_premises.md

```diff
@@ -165,7 +165,8 @@ We're prefixing each item with how we'll refer to it further on.
 
 1. `OS_ENDPOINT`: The access endpoint.
    For example, MinIO's endpoints are usually something like `minio-server:9000`. 
-   Don't begin it with the protocol; it's an endpoint, not an url.
+   Don't begin it with the protocol; it's an endpoint, not an url. Also, check if your local instance of e.g. MinIO is using SSL/TLS
```

Comment:
By local instance, are you referring to the client running on the local machine?

Reply:
"Local" is of course not very precise, well spotted. I was referring to the on-prem deployment of MinIO in the k8s cluster. A better formulation might be "Also, check if the object storage deployment (e.g. MinIO) in your cluster uses SSL/TLS ..."